roundabout-core/src/main/java/at/fhv/itm3/s2/roundabout/entity/StreetSection.java

```diff
@@ -58,8 +58,8 @@ public void updateAllCarsPositions() {
         throw new NotImplementedException();
     }
 
-    public boolean isFirstCarOnExitPoint() {
-        throw new NotImplementedException();
+    public boolean isFirstCarOnExitPoint(ICar car) {
+        return Math.floor(carPositions.get(car) * 100) == 0.0; // also almost zero
```

Comment:
first update all car positions, like written in issue

Reply:
use this.getFirstCar() method instead of car

Reply:
use this.getCarPaositions() getter instead of direct use of attribute

Reply:
more information about comparing 2 doubles: https://stackoverflow.com/questions/25160375/comparing-double-values-for-equality-in-java